src/redux/constants/actionTypes.js

```diff
@@ -51,3 +51,13 @@ export const UPDATE_ITEM_DETAIL = "UPDATE_ITEM_DETAIL";
 export const OPEN_REVIEW_MODAL = "OPEN_REVIEW_MODAL";
 export const ADD_REVIEW = "ADD_REVIEW";
 export const SET_LOADING = "SET_LOADING";
+
+/**
+ * Notification Action Types
+ **/
+export const GET_NOTIFICATIONS = "GET_NOTIFICATIONS";
+export const ADD_NOTIFICATION = "ADD_NOTIFICATION";
```

Comment:
@Cainuse @abid-salahi 
Not sure what the workflow for notifications will be like on the backend. When I fire off an action on an offer (so for example, I click accept an offer), will backend handle the notification stuff for me? Or will I need to handle the notifications through the frontend?

Reply:
I guess backend will take care of it. will remove this action.